### ISSUE TITLE
Moving objects bug: Can't move selections of objects that include a group

### ISSUE THREAD

**Issue by mkutmon** [PathVisio/pathvisio#14]
_Reported by Kristina:_

It seems impossible to move any selection of objects that include a group of data nodes. By move, I mean select then click and drag to move or by using arrow keys.

To test, create a group of two data nodes and select it together with a label or any other object.

**Comment by DeniseSl22** [comment]
Yes, I'm also experiencing this; we shoudl check if we can fix it for PV4...
